integration_tests/dbt_project.yml

```diff
@@ -14,6 +14,11 @@ vars:
     ga4_export_event_identifier: "event_data"
     key_events: ['purchase']
 
+models:
+  ga4_export:
+    staging:
+      +materialized: table
```

Comment:
Hi @fivetran-joemarkiewicz , what do you mean by incomplete seed data? As in we should add source_relation as a field to the seed data?

Reply:
Incomplete as in the constant expression issue is not just being caused by the `source_relation` field, but the others as well since we only have a few records in the seed data. We wouldn't add source_relation to the seed data, but this error likely wouldn't occur when using a complete dataset where there are more than 4 records.

Reply:
Gotcha. And what did you mean by resolving via materialization strategy?